xdsl/utils/parse_pipeline.py

```diff
@@ -26,6 +26,7 @@ class Kind(Enum):
         STRING_LIT = object()
         MLIR_PIPELINE = object()
         COMMA = ","
+        DOT = "."
```

Comment:
is this still necessary with the quotes fix?

Reply:
Ok done